aiomqtt/client.py

```diff
@@ -549,6 +549,7 @@ def _force_disconnect(self) -> None:
     @_outgoing_call
     async def subscribe(  # noqa: PLR0913
         self,
+        /,
```

Comment:
This is technically a breaking change.

This goes for all public methods (including `subscribe`, `unsubscribe`, `publish`).

Before this PR, our users could do:

```python
client.subscribe(topic="floors/first")
```

After this PR, the code above gives an error.

To avoid breaking stuff, I think that we should postpone this change until a 2.0 release. :)

Does it make sense?

Reply:
I tested this just now and the example works for me. It fails when I put the `/` _after_ the `topic` parameter (making `topic` positional-only). I've never used this feature before, however, so could be that I made an error somewhere!

I mainly added it because mypy wanted it. I'm ok with ignoring mypy's concerns as well and postponing that change, like you propose 🙂

Reply:
You are completely right. :+1:  My bad. I misunderstood the semantics of the `/` operator. :sweat_smile:

Reply:
No worries, thanks for double-checking nonetheless!